File: src/main/java/kr/co/lunatalk/domain/cartitem/repository/CartItemRepository.java
package kr.co.lunatalk.domain.cartitem.repository;

import kr.co.lunatalk.domain.cartitem.domain.CartItem;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface CartItemRepository extends JpaRepository<CartItem, Long>, CartItemRepositoryCustom {

}

File: src/main/kotlin/kr/co/lunatalk/domain/cartitem/repository/CartItemRepository.kt
package kr.co.lunatalk.domain.cartitem.repository

import kr.co.lunatalk.domain.cartitem.domain.CartItem
import org.springframework.data.jpa.repository.JpaRepository

interface CartItemRepository : JpaRepository<CartItem, Long>, CartItemRepositoryCustom {

    fun deleteByMemberIdAndProductId(memberId: Long, productId: Long)
}
